apps/pie-microsite/src/__tests__/utilities/helpers.test.js

```diff
@@ -49,4 +49,18 @@ describe('helpers.js', () => {
             });
         });
     });
+
+    describe('numberHelpers', () => {
+        describe('isNumber', () => {
+            [['1', true], ['fooBarBaz', false]].forEach(example => {
+                it('correctly checks whether string can convert to number', () => {
```

Comment:
Out of interest – will this function pass true if you pass an actual number to it?

Main reason I ask is that the name of the function implies that it'd return true in that scenario. If it does pass, then the function name is all good, but if it doesn't for some reason, I'd rename the function to be more string specific.

I'd add a test here for passing in a number either way (so just `10` or something), as it'd be good to have that covered.

Reply:
just checked and it works with a number too - I've added the test and will leave the function name as is 😄